gengokit/httptransport/templates/server.go

```diff
@@ -173,9 +180,12 @@ func (h httpError) Headers() http.Header {
 // EncodeHTTPGenericResponse is a transport/http.EncodeResponseFunc that encodes
 // the response as JSON to the response writer. Primarily useful in a server.
 func EncodeHTTPGenericResponse(_ context.Context, w http.ResponseWriter, response interface{}) error {
-	encoder := json.NewEncoder(w)
-	encoder.SetEscapeHTML(false)
```

Comment:
We are a little concerned about this. Could you point us at documentation that jsonpb, does *not* do html escaping? 

See here https://godoc.org/encoding/json#Encoder.SetEscapeHTML. We had a problem with this in production before.

We will be testing this in our staging environment for one of products here at Tune here, as we do not have tests for the HTML escaping in json. I believe that I'll be able to do that before the end of the week. If you would like to add a test for HTML characters being escaped in json, it would be highly appreciated and give strong confidence for merging this.